Tasks/Mardoyan/Task6/Task6_3.py

```diff
@@ -0,0 +1,16 @@
+c = 0
+def count_func(n,a):
+    global c         #Знаю,что глобал лучше не юзать,но не нашел выхода другого.Видел в занятии про nonlocal,но там было 2 функции
```

Comment:
yeah-yeah-yeah, check solutions of your colleagues, it can be done without global